src/callbacks_step/glm_speed.jl

```diff
@@ -40,15 +44,16 @@ function Base.show(io::IO, ::MIME"text/plain",
         setup = [
             "GLM wave speed scaling" => glm_speed_callback.glm_scale,
             "Expected CFL number" => glm_speed_callback.cfl,
+            "Coupled semis indices" => glm_speed_callback.semi_indices,
```

Comment:
```suggestion
            "Selected semidiscretizations" => glm_speed_callback.semi_indices,
```
For clarity? Not 100% sure if it's an improvement, but "coupled semis indices" sounded not very precise

Reply:
Good idea. Since these are the indices of the semidiscretizations for which to compute the GLM speed.